src/styles/index.scss

```diff
@@ -1,3 +1,146 @@
+$widthOfRating: calc(100% - 32px);
+$grayColor: #616070;
+$blackColor: #060b35;
+$whiteColor: #fff;
+$whiteShade: #f3f3f3;
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $whiteShade;
+  border-radius: 5px;
+  background-color: $whiteColor;
+
+  &__img {
+    margin: 32px 19px 40px;
+    height: 134px;
+
+    .img-content {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__content {
+    margin: 0 16px;
+
+    &-title {
+      color: $blackColor;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      font-family: "Roboto", sans-serif;
+    }
+
+    &-prcode {
+      color: $grayColor;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      font-family: "Roboto", sans-serif;
+      margin-top: 4px;
+    }
+  }
+
+  &__rating {
+    width: $widthOfRating;
+    margin: 16px 16px 0;
+    max-height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .stars {
+      display: flex;
+
+      &__star {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 16px;
+        width: 16px;
+        margin-right: 4px;
+        background-image: url(../images/star.svg);
+        background-repeat: no-repeat;
+        background-position: center;
+      }
+
+      :nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+
+    .review {
+      display: flex;
+      align-items: center;
+
+      &__content {
+        color: $blackColor;
+        font-family: "Roboto", sans-serif;
+        font-weight: 400;
+        font-size: 10px;
+        text-align: right;
+      }
+    }
+  }
+
+  &__price {
+    margin: 24px 16px 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-title {
+      color: $grayColor;
+      font-family: "Roboto", sans-serif;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-number {
+      color: $blackColor;
+      font-family: "Roboto", sans-serif;
+      font-size: 16px;
+      font-weight: 700;
+      text-align: right;
+      line-height: 18px;
+    }
+  }
+
+  &__submit {
+    margin: 16px;
```

Comment:
you give horizontal margins almost everywhere. better to set just vertical ones and give padding for the whole card instead